### ISSUE TITLE
Druid build fails with git command '-no-show-signature'

### ISSUE THREAD

**Issue by basapuram-kumar** [apache/druid#15124]
Building Druid with the below command on master/v27.0 branches:
```
mvn clean package  install -DskipTests   -Pdist -Prat
````

the build  failed with the below error.

```
[ERROR] Failed to execute goal pl.project13.maven:git-commit-id-plugin:4.9.10:revision (default) on project druid: Could not complete Mojo execution...: pl.project13.core.NativeGitProvider$NativeCommandException: Git command exited with invalid status [128]: directory: `/root/bassa/apache/druid`, command: `git log -1 --pretty=format:%ct --no-show-signature HEAD`, stdout: ``, stderr: `fatal: unrecognized argument: --no-show-signature`
```

Solution:
comment the plugin from https://github.com/apache/druid/blob/master/pom.xml#L1569 to https://github.com/apache/druid/blob/master/pom.xml#L1607

Ex: 
```
            <!--plugin>
                <groupId>pl.project13.maven</groupId>
                <artifactId>git-commit-id-plugin</artifactId>
                <version>4.9.10</version>
                <executions>
                    <execution>
                        <goals>
                            <goal>revision</goal>
                        </goals>
                    </execution>
                </executions>
                <configuration>
                    <dotGitDirectory>${project.basedir}/.git</dotGitDirectory>
                    <dateFormatTimeZone>Etc/UTC</dateFormatTimeZone>
                    <skipPoms>false</skipPoms>
                    <format>json</format>
                    <generateGitPropertiesFile>true</generateGitPropertiesFile>
                    <generateGitPropertiesFilename>${project.build.directory}/git.version</generateGitPropertiesFilename>
                    <failOnNoGitDirectory>false</failOnNoGitDirectory>
                    <useNativeGit>true</useNativeGit>
                    <excludeProperties>
                        <excludeProperty>git.build.user.email</excludeProperty>
                        <excludeProperty>git.build.host</excludeProperty>
                        <excludeProperty>git.commit.id.describe-short</excludeProperty>
                        <excludeProperty>git.commit.user.*</excludeProperty>
                        <excludeProperty>git.commit.message.*</excludeProperty>
                        <excludeProperty>git.closest.tag.*</excludeProperty>
                        <excludeProperty>git.commit.id.abbrev</excludeProperty>
                        <excludeProperty>git.dirty</excludeProperty>
                    </excludeProperties>
                    <gitDescribe>
                        <skip>false</skip>
                        <always>true</always>
                        <abbrev>7</abbrev>
                        <dirty>-dirty</dirty>
                        <forceLongFormat>true</forceLongFormat>
                    </gitDescribe>
                </configuration>
            </plugin-->

```

After commenting the above line, build crosses this error stage.

**Comment by AlbericByte** [comment]
@FrankChen021 and @basapuram-kumar please help and have a look. Thanks for you time.

**Comment by FrankChen021** [comment]
@AlbericByte I can't remeber that I suffered this problem before, let me confirm this problem first.

**Comment by basapuram-kumar** [comment]
@AlbericByte I will  run the build with with this fix, and let you know. Thanks.

**Comment by AlbericByte** [comment]
@FrankChen021 yes, if i add -Pdist -Prat parameters, I can reproduce this issue everytime.

**Comment by FrankChen021** [comment]
Note: the original problem described in this issue is not actually resolved by #15530
The original problem is not re-produced at my side, @basapuram-kumar if you still suffer the problem on the latest master branch, can re-open this issue again.
